tests/phpunit/tests/hooks/addFilter.php

```diff
@@ -123,9 +130,11 @@ public function test_readd_filter_with_different_priority() {
 		$accepted_args = 2;
 
 		$hook->add_filter( $hook_name, $callback, $priority, $accepted_args );
+		$this->check_priority_exists( $hook, $priority );
```

Comment:
Should we be more specific to match the priority's method within the hook? Maybe something like:

```suggestion
		$hook->add_filter( $hook_name, $callback, $priority, $accepted_args );
		$this->check_priority( $hook, $priority, $callback );
```

And then instead of `assertContains`, use something like `assertSame`.

Reply:
The array contains a list of priorities and not callbacks.

Reply:
Ah, right. In that case, maybe we can ensure the order of the list is maintained?

Trying to see if we can further strengthen the assertion check.

Reply:
@bor0 I have improve unit tests.